lib/elixir/pages/Guards.md

```diff
@@ -55,6 +55,8 @@ For reference, the following is a comprehensive list of all expressions allowed
     * `bsr/1` or the `>>>` operator
     * `bxor/2` or the `^^^` operator
 
+Macros constructed out of any combination of the above guards are also valid guards - e.g., `Integer.is_even/1`. (Such macros usually include documentation such as `Allowed in guard clauses`).
```

Comment:
I would:

* point to the section "Defining custom guards expressions" below
* not use parentheses to say that usually that usually these macros include documentation that states they're allowed in guard clauses